src/test/java/seedu/address/model/AddressBookTest.java

```diff
@@ -42,8 +42,8 @@ public void resetData_withValidReadOnlyAddressBook_replacesData() {
     }
 
     @Test
-    public void resetData_withDuplicatePersons_throwsDuplicatePersonException() {
-        // Two persons with the same identity fields
+    public void resetData_withDuplicatePatients_throwsDuplicatePatientException() {
+        // Two patient with the same identity fields
```

Comment:
I think it would be `patients` here. Very minor grammatical issue though 😄

Reply:
okayy, ill change that real quick